.github/workflows/checkstyle.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build
```

Comment:
Change the name of the workflow to "Check style" or something

Reply:
Just to confirm:
at the moment we have this: Inside the "Build" we have "Checkstyle"
![image](https://github.com/user-attachments/assets/b5188ed8-e24b-4dd5-9194-44155435a4e2)

Should it be like this?
![image](https://github.com/user-attachments/assets/14b8a886-cefd-4e89-a241-b9038d7400f5)

Reply:
Yes. If you look at the actions page, it looks like this right now, which is confusing
![image](https://github.com/user-attachments/assets/5d62bc85-2434-4c3b-b3fc-b17ab5edc80f)